dask/dataframe/tests/test_rolling.py

```diff
@@ -86,11 +92,36 @@ def test_rolling_funtions_dataframe():
     rolling_functions_tests(df, ddf)
 
 
-def test_rolling_dataframe():
-    df = pd.DataFrame({'a': np.random.randn(25).cumsum(),
-                       'b': np.random.randint(100, size=(25,))})
-    ddf = dd.from_pandas(df, 3)
-    basic_rolling_tests(df, ddf)
+@pytest.mark.parametrize('npartitions', [1, 2, 3])
+@pytest.mark.parametrize('method,args', [
+    pytest.mark.xfail(('count', ())), # pandas ignores axis arg, causing some failures
```

Comment:
This xfail is broader than we need, but I don't know if pytest has a non-ridiculous way to narrow it the way we would like.